tests/baselines/reference/nodeModulesTripleSlashReferenceModeDeclarationEmit3(module=node16).js

```diff
@@ -25,15 +25,54 @@ declare global {
     "type": "module"
 }
 //// [index.ts]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" resolution-mode="require" preserve="true" />
 export interface LocalInterface extends RequireInterface {}
 
 //// [index.js]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" resolution-mode="require" preserve="true" />
 export {};
 
 
 //// [index.d.ts]
-/// <reference types="pkg" resolution-mode="require"/>
+/// <reference types="pkg" preserve="true" />
 export interface LocalInterface extends RequireInterface {
 }
+
+
+//// [DtsFileErrors]
+
+
+out/index.d.ts(2,41): error TS2304: Cannot find name 'RequireInterface'.

```

Comment:
This test is super weird and I'm not finding any way to make this error go away, even though `pkg` is preserved?